### PR TITLE
add singint signal support

### PR DESCRIPTION
**Pull request by tanhe123**
1. fix windows not exit bug after function invoked.
2. add ctrl + c signal support for canceling running functions.
3. fix windows could not pull image bug.

### CODE REVIEW

File: lib/docker.js [hunk 1]
```diff
@@ -103,24 +160,76 @@ function findDockerImage(runtime) {
...
-    function onProgress(event) {
-      debug(event);
-      process.stdout.write('.');
-    }
+  // below code will occur problem on windows when using pkg packed binary version
+  // error is 'context canceled'
 
-    docker.modem.followProgress(stream, onFinished, onProgress);
-  });
+  ////// begin
```
Comment (vangie) [inline lib/docker.js]: 这一大段注释代码还有意义么？

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alicloud/fun",
-  "version": "2.6.1",
+  "version": "2.7.0",
```
Comment (vangie) [inline package.json]: 没有加新的 feature 吧，2.6.2

Comment (tanhe123) [reply package.json]: 我把 ctrl + c 取消函数运行算 feature 了

File: lib/docker.js (hunk 1 above)
Comment (tanhe123) [reply lib/docker.js]: removed

**Pull request merged**